containers/Dockerfile.base-spack

```diff
@@ -78,7 +81,15 @@ SHELL ["/bin/bash", "-c"]
 # Install spack
 RUN git clone -c feature.manyFiles=true https://github.com/spack/spack.git ${SPACK_ROOT} --branch ${SPACK_REPO_VERSION} --single-branch --depth=1
 
-RUN sed -i 's/granularity: microarchitectures/granularity: generic/' ${SPACK_ROOT}/etc/spack/defaults/concretizer.yaml
+# Install ACCESS-NRI's spack_config repo
+RUN git clone  https://github.com/ACCESS-NRI/spack_config.git ${SPACK_CONFIG_REPO_ROOT} --branch ${SPACK_CONFIG_REPO_VERSION}
```

Comment:
```suggestion
RUN git clone  https://github.com/ACCESS-NRI/spack_config.git ${SPACK_CONFIG_REPO_ROOT} --branch ${SPACK_CONFIG_REPO_VERSION} --single-branch --depth=1
```

Reply:
In the Docker image, I need to be able to test other branches of spack_config. Remember, it's a dev image for me.

Reply:
Ah, noted. Removed suggestion.